### ISSUE TITLE
Enable styles doesn't work consistently

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1347]
There are some instances when the "Enable styles" button cannot fully restore styles, and the page must be reloaded via a browser refresh to return it to its original state. This seems to be particularly problematic on sites with sliders or carousels, or perhaps inline styles?

You can see this by going to the following websites, viewing the page on the front end, disabling styles in Accessibility Checker, and then enabling them again.
- Our website (block editor)
- Driscoll Children's (Oxygen)
- Highland (Block editor, also TEC archive page)
- Road Warrior Creative home page (ACF template)

I don't think this is a high-priority issue, as we haven't heard any user complaints about it, and refreshing the browser is trivial, so it is not a blocker to our own work. However, I'm opening an issue since that was mentioned in team meeting this morning.
